bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java

```diff
@@ -74,162 +74,166 @@ public JdbcConfiguration(Map<Object, Object> configuration) {
+        }
 
-            String dd = (String) configuration.get("numberDecimalcount");
-            if (StringUtils.isNotBlank(dd) && StringUtils.isNumeric(dd)) {
-                numberDecimalcount = Integer.parseInt(dd);
-                logger.debug("JDBC::updateConfig: numberDecimalcount={}", numberDecimalcount);
-            }
+        String np = (String) configuration.get("tableNamePrefix");
+        if (StringUtils.isNotBlank(np)) {
+            tableNamePrefix = np;
+            logger.debug("JDBC::updateConfig: tableNamePrefix={}", tableNamePrefix);
+        }
 
-            String rn = (String) configuration.get("tableUseRealItemNames");
-            if (StringUtils.isNotBlank(rn)) {
-                tableUseRealItemNames = "true".equals(rn) ? Boolean.parseBoolean(rn) : false;
-                logger.debug("JDBC::updateConfig: tableUseRealItemNames={}", tableUseRealItemNames);
-            }
+        String dd = (String) configuration.get("numberDecimalcount");
+        if (StringUtils.isNotBlank(dd) && StringUtils.isNumeric(dd)) {
+            numberDecimalcount = Integer.parseInt(dd);
+            logger.debug("JDBC::updateConfig: numberDecimalcount={}", numberDecimalcount);
+        }
 
-            String td = (String) configuration.get("tableIdDigitCount");
-            if (StringUtils.isNotBlank(td) && StringUtils.isNumeric(td)) {
-                tableIdDigitCount = Integer.parseInt(td);
-                logger.debug("JDBC::updateConfig: tableIdDigitCount={}", tableIdDigitCount);
-            }
+        String rn = (String) configuration.get("tableUseRealItemNames");
+        if (StringUtils.isNotBlank(rn)) {
+            tableUseRealItemNames = "true".equals(rn) ? Boolean.parseBoolean(rn) : false;
+            logger.debug("JDBC::updateConfig: tableUseRealItemNames={}", tableUseRealItemNames);
+        }
 
-            String rt = (String) configuration.get("rebuildTableNames");
-            if (StringUtils.isNotBlank(rt)) {
-                rebuildTableNames = "true".equals(rt) ? Boolean.parseBoolean(rt) : false;
-                logger.debug("JDBC::updateConfig: rebuildTableNames={}", rebuildTableNames);
-            }
+        String td = (String) configuration.get("tableIdDigitCount");
+        if (StringUtils.isNotBlank(td) && StringUtils.isNumeric(td)) {
+            tableIdDigitCount = Integer.parseInt(td);
+            logger.debug("JDBC::updateConfig: tableIdDigitCount={}", tableIdDigitCount);
+        }
 
-            // undocumented
-            String ac = (String) configuration.get("maximumPoolSize");
-            if (StringUtils.isNotBlank(ac)) {
-                dBDAO.databaseProps.setProperty("maximumPoolSize", ac);
-            }
+        String rt = (String) configuration.get("rebuildTableNames");
+        if (StringUtils.isNotBlank(rt)) {
+            rebuildTableNames = "true".equals(rt) ? Boolean.parseBoolean(rt) : false;
```

Comment:
This could be simplified to just `rebuildTableNames = Boolean.parseBoolen(rt);` and allows case-insensitive checking which can't hurt.